pkg/action/install.go

```diff
@@ -817,20 +817,26 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 	name = strings.TrimSpace(name)
 	version := strings.TrimSpace(c.Version)
 
-	if _, err := os.Stat(name); err == nil {
-		abs, err := filepath.Abs(name)
-		if err != nil {
-			return abs, err
-		}
-		if c.Verify {
-			if _, err := downloader.VerifyChart(abs, abs+".prov", c.Keyring); err != nil {
-				return "", err
+	if c.RepoURL == "" {
+		if _, err := os.Stat(name); err == nil {
+			abs, err := filepath.Abs(name)
+			if err != nil {
+				return abs, err
+			}
+			if c.Verify {
+				if _, err := downloader.VerifyChart(abs, abs+".prov", c.Keyring); err != nil {
+					return "", err
+				}
+				return abs, nil
 			}
+			if filepath.IsAbs(name) || strings.HasPrefix(name, ".") {
```

Comment:
Why is it wanted/needed to check if name is an absolute or hidden file here?

```
helm create foo
./bin/helm template `realpath foo`
Error: path "/Users/gjenkins8/Source/helm/helm/foo" not found
```

This seems to be wrong behavior?